akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -332,8 +283,7 @@ private[akka] object Running {
           this // it will be stopped by the first WriteMessageFailure message; not applying side effects
 
         case _ =>
-          // ignore all other messages, since they relate to recovery handling which we're not dealing with in Running phase
-          Behaviors.unhandled
+          onDeleteEventsJournalResponse(response)
```

Comment:
Feels a bit weird with a catch-all and then assume response is delete event response (and then another catchall in the end of that) - setting ourselves up for foot shooting?

Reply:
Alternative would be to compose partial functions, not sure that is any better. This `onDeleteEventsJournalResponse` is shared and used from several places.

I could try to make `onDeleteEventsJournalResponse` a Behavior and compose with `orElse`. I'd rather do that in a separate follow up https://github.com/akka/akka/issues/26597